templates/core/build.html

```diff
@@ -175,15 +191,15 @@ <h4>Your chosen snaps:</h4>
   <section class="p-strip--light is-shallow">
     <div class="row">
       <div class="col-8">
-        <form method="POST">
+        <form method="POST" id="build-form">
           <p>We’ll email you at {{ openid.email }} once the build is ready.</p>
           <p>
             <input class="mktoField" value="yes" id="canonicalUpdatesOptIn" name="canonicalUpdatesOptIn" type="checkbox">
             <label class="mktoLabel mktoHasWidth" for="canonicalUpdatesOptIn">I agree to receive information about Canonical’s products and services.</label>
           </p>
-          <input type="hidden" name="board" value="cm3" />
-          <input type="hidden" name="system" value="core16" />
-          <input type="hidden" name="snaps" value="toto,code,toto-fran" />
+          <input type="hidden" name="board" value="" />
```

Comment:
Weren't you gonna make these text fields / radio buttons or something?

Reply:
No, I think its easier to update if you just add a new card instead of both the card and the value in a radio button.

Reply:
Oh so there isn't a JS-less option?

Reply:
^ ?

Reply:
I think I need a follow up card to make this UI work without JS as I need to work out the snaps input too.